src/Lykke.Service.B2c2Adapter/Services/OrderBooksService.cs

```diff
@@ -192,9 +272,38 @@ private static IEnumerable<OrderBookItem> GetOrderBookItems(IEnumerable<Quantity
             return result;
         }
 
+        private void SetHelthCheck(PriceMessage result)
+        {
+            if (_healthCheck.TryGetValue(result.Instrument, out var counter))
```

Comment:
Can you use AddOrUpdate method? What if another thread adds value between calls.

Reply:
How? There is different behavior for 2 cases.

Reply:
Didn't understand note about another thread